### PR TITLE
add standard

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
I thought I'd make a pass of standard before fixing the serialize stuff.

File: deferred-iterator.js [hunk 1]
```diff
@@ -13,15 +12,14 @@ function DeferredIterator (options) {
 util.inherits(DeferredIterator, AbstractIterator)
 
 DeferredIterator.prototype.setDb = function (db) {
-  var it = this._iterator = db.iterator(this._options)
+  var it = (this._iterator = db.iterator(this._options))
```
Comment (vweevers) [inline deferred-iterator.js]: This was probably added by `prettier`? The parentheses are a bit weird

Comment (ralphtheninja) [reply deferred-iterator.js]: Agreed! Fixing.

**Comment by vweevers** [comment]
@ralphtheninja I noticed the tiny logo doesn't work on npm, so random idea:

# deferred-leveldown

> A mock `abstract-leveldown` implementation that queues operations while a real `abstract-leveldown` instance is being opened.

![level badge][level-badge] [Build Status](https://travis-ci.org/Level/deferred-leveldown) [Greenkeeper badge](https://greenkeeper.io/) [JavaScript Style Guide](https://standardjs.com)

[level-badge]: https://img.shields.io/badge/-level-brightgreen.svg?colorA=000000&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8%2BPHN2ZyB3aWR0aD0iMTg4cHgiIGhlaWdodD0iMjQycHgiIHZpZXdCb3g9IjAgMCAxODggMjQyIiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHhtbG5zOnNrZXRjaD0iaHR0cDovL3d3dy5ib2hlbWlhbmNvZGluZy5jb20vc2tldGNoL25zIj4gICAgPHRpdGxlPlVudGl0bGVkIDEzPC90aXRsZT4gICAgPGRlc2NyaXB0aW9uPkNyZWF0ZWQgd2l0aCBTa2V0Y2ggKGh0dHA6Ly93d3cuYm9oZW1pYW5jb2RpbmcuY29tL3NrZXRjaCk8L2Rlc2NyaXB0aW9uPiAgICA8ZGVmcz48L2RlZnM%2BICAgIDxnIGlkPSJQYWdlLTEiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIHNrZXRjaDp0eXBlPSJNU1BhZ2UiPiAgICAgICAgPHBhdGggZD0iTTk0LDU0IEwwLDEwMC40MTAwNDIgTDAsMTk0LjgwMzM0NyBMOTQsMjQyIEwxODgsMTk0LjgwMzM0NyBMMTg4LDEwMC40MTAwNDIgTDk0LDU0IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iI0Q0RUI5NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDM5IEwwLDg1LjQxMDA0MTggTDAsMTc5LjgwMzM0NyBMOTQsMjI3IEwxODgsMTc5LjgwMzM0NyBMMTg4LDg1LjQxMDA0MTggTDk0LDM5IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iIzk2REM3NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDEgTDAsNDcuNDEwMDQxOCBMMCwxNDEuODAzMzQ3IEw5NCwxODkgTDE4OCwxNDEuODAzMzQ3IEwxODgsNDcuNDEwMDQxOCBMOTQsMSBaIiBpZD0iUmVjdGFuZ2xlLTEiIG9wYWNpdHk9IjAuNTUyODQwOTA5IiBmaWxsPSIjMzE3MzQyIiBza2V0Y2g6dHlwZT0iTVNTaGFwZUdyb3VwIj48L3BhdGg%2BICAgICAgICA8cGF0aCBkPSJNOTQsOTEuMTc3Nzc5MSBMMCw0NyBMMCwxNDEuNjY2NjY3IEw5NCwxODkgTDE4OCwxNDEuNjY2NjY3IEwxODgsNDcgTDk0LDkxLjE3Nzc3OTEgWiIgaWQ9IlJlY3RhbmdsZS0xIiBvcGFjaXR5PSIwLjU1Mjg0MDkwOSIgZmlsbD0iIzM0OTU0QyIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICA8L2c%2BPC9zdmc%2B

**Comment by ralphtheninja** [comment]
@vweevers :heart: Love it!

**Comment by ralphtheninja** [comment]
@vweevers I also like (from memdown)

memdown-downloads

I think we should use it on all modules instead of the bigger version:

leveldown-downloads

**Comment by vweevers** [comment]
For the level badge, we can either use shields.io (with a bit of a hack, see below) or copy the resulting SVG to the website.

```md
![level badge][level-badge]

[level-badge]: https://img.shields.io/badge/-level-brightgreen.svg?colorA=000000&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8%2BPHN2ZyB3aWR0aD0iMTg4cHgiIGhlaWdodD0iMjQycHgiIHZpZXdCb3g9IjAgMCAxODggMjQyIiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHhtbG5zOnNrZXRjaD0iaHR0cDovL3d3dy5ib2hlbWlhbmNvZGluZy5jb20vc2tldGNoL25zIj4gICAgPHRpdGxlPlVudGl0bGVkIDEzPC90aXRsZT4gICAgPGRlc2NyaXB0aW9uPkNyZWF0ZWQgd2l0aCBTa2V0Y2ggKGh0dHA6Ly93d3cuYm9oZW1pYW5jb2RpbmcuY29tL3NrZXRjaCk8L2Rlc2NyaXB0aW9uPiAgICA8ZGVmcz48L2RlZnM%2BICAgIDxnIGlkPSJQYWdlLTEiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIHNrZXRjaDp0eXBlPSJNU1BhZ2UiPiAgICAgICAgPHBhdGggZD0iTTk0LDU0IEwwLDEwMC40MTAwNDIgTDAsMTk0LjgwMzM0NyBMOTQsMjQyIEwxODgsMTk0LjgwMzM0NyBMMTg4LDEwMC40MTAwNDIgTDk0LDU0IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iI0Q0RUI5NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDM5IEwwLDg1LjQxMDA0MTggTDAsMTc5LjgwMzM0NyBMOTQsMjI3IEwxODgsMTc5LjgwMzM0NyBMMTg4LDg1LjQxMDA0MTggTDk0LDM5IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iIzk2REM3NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDEgTDAsNDcuNDEwMDQxOCBMMCwxNDEuODAzMzQ3IEw5NCwxODkgTDE4OCwxNDEuODAzMzQ3IEwxODgsNDcuNDEwMDQxOCBMOTQsMSBaIiBpZD0iUmVjdGFuZ2xlLTEiIG9wYWNpdHk9IjAuNTUyODQwOTA5IiBmaWxsPSIjMzE3MzQyIiBza2V0Y2g6dHlwZT0iTVNTaGFwZUdyb3VwIj48L3BhdGg%2BICAgICAgICA8cGF0aCBkPSJNOTQsOTEuMTc3Nzc5MSBMMCw0NyBMMCwxNDEuNjY2NjY3IEw5NCwxODkgTDE4OCwxNDEuNjY2NjY3IEwxODgsNDcgTDk0LDkxLjE3Nzc3OTEgWiIgaWQ9IlJlY3RhbmdsZS0xIiBvcGFjaXR5PSIwLjU1Mjg0MDkwOSIgZmlsbD0iIzM0OTU0QyIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICA8L2c%2BPC9zdmc%2B
```

**Comment by ralphtheninja** [comment]
Cool. Do you think we should try to have some thought behind the badges? I.e. what to actually use, in what order etc. I've added the Greenkeeper stuff but now afterwards I'm not sure why we really need that one. Do people care if we use Greenkeeper?

**Comment by ralphtheninja** [comment]
We could use https://github.com/Level/community as repo for more meta or level related issues

**Comment by vweevers** [comment]
> We could use https://github.com/Level/community as repo for more meta or level related issues

👍 I was looking for something like that :)

**Comment by ralphtheninja** [comment]
2.0.2

**Pull request merged**